WcaOnRails/spec/controllers/panel_controller_spec.rb

```diff
@@ -13,11 +13,6 @@
       get :pending_claims_for_subordinate_delegates
       expect(response.status).to eq 200
     end
-
-    it "can't view the seniors list" do
```

Comment:
Why are you dropping these tests without replacement?

Reply:
Actually the seniors panel (/panel/seniors) doesn't exist anymore. This was only accessible to board and now it's moved to Board panel. Sorry that I missed to remove it from routes.rb and panel_controller.rb. I've removed them now.